Libraries/LibJS/Tests/debugger-statement.js

```diff
@@ -1,9 +1,3 @@
-load("test-common.js");
-
-try {
-    debugger;
-
-    console.log("PASS");
-} catch (e) {
-    console.log("FAIL: " + e);
-}
+test("debugger keyword", () => {
+    expect("debugger").toCompile();
```

Comment:
I feel like we should distinguish between "parses without errors" and "runs without errors" - previously it would actually execute the statement. Anything against keeping that behaviour for this test?

OTOH `debugger` is somewhat special and being able to execute it without any effects might be counter-intuitive, so I understand if this was a concious decision :)